<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMX Selection Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#31363B;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:#31363B;
}
QTabWidget{
background-color:#31363B;
border: 0;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Raspbery Pi DMX Mode</string>
     </attribute>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: white;

}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="raspberryPiPasswordInput">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{color:white;}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="mainLabel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: white;
border: 1px solid white;
border-radius: 5px;
text-align: center;

}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Raspberry pi log in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="raspberryPiSubmitButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Computer DMX Mode</string>
     </attribute>
     <widget class="QRadioButton" name="windowsRadio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{color:white;}</string>
      </property>
      <property name="text">
       <string>Windows (COM)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="linuxRadio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{color:white;}</string>
      </property>
      <property name="text">
       <string>Linux /dev/ttyUSB</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="portSelectionInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{color:white;}</string>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QPushButton" name="computerDMXSubmitButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{color:white;}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Use a USB cable connected to this device to control the lighting rig. Enter the port number:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="testPortButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Test Port</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Run Without DMX</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{color:white;}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Run the program without sending any data to the lighting rig. This can be used for creating rigs without actually being at the venue of the lighting rig.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="runWithoutDMXButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
